app/views/renders/_user.html.erb

```diff
@@ -0,0 +1,12 @@
+<li class="card-wrap">
+  <%= link_to user do %>
+    <%= image_tag user.photo, class:'card-image', alt: 'photo' %>
+  <% end %>
+  <div class="card">
+    <h2 class="card-title">
+      <%= user.name %>
+    </h2>
+    <p class="card-para">number of posts:
+      <%= user.posts_counter || 0 %>
+  </div>
```

Comment:
Please. add a link to this to make the user profile easily accessible not just on the image.